src/storage/config.rs

```diff
@@ -225,3 +242,107 @@ impl BlockCacheConfig {
         None
     }
 }
+
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Configuration)]
+#[serde(default)]
+#[serde(rename_all = "kebab-case")]
+pub struct IORateLimitConfig {
+    pub max_bytes_per_sec: ReadableSize,
```

Comment:
I forget if I commented on this, but can we make it a `Option<ReadableSize>`, to avoid using "0MB/s" to denote no rate limiter?

Reply:
Do you mean an additional representation for no rate limit? Or we should also raise an error for using 0MB?